docs/uml/main/InputDatamodelConcept.puml

```diff
@@ -36,11 +36,20 @@ abstract Class ConnectorInput {
 }
 ConnectorInput --|> AssetInput
 
-abstract Class AssetCharacteristicsInput {
-+ type: String
+abstract Class AssetCharacteristicsInput<T extends AssetTypeInput> {
+    - type: T
+    - Characteristic: String
 }
 AssetCharacteristicsInput --|> InputEntity
 
+Class WecCharacteristicInput {
+}
+WecCharacteristicInput --|> AssetCharacteristicsInput: <<bind>>T::WecTypeInput
```

Comment:
Would you mind explaining me what `<<bind>>` means, please?

Reply:
I'm not an UML expert, too. It should underline, that the implementation binds the generic type of `AssetCharacteristicInput` to `WecTypeInput`.